app/livechat/server/api/lib/livechat.js

```diff
@@ -76,7 +76,8 @@ export function getRoom({ guest, rid, roomInfo, agent }) {
 		ts: new Date(),
 	};
 
-	return Livechat.getRoom(guest, message, roomInfo, agent);
+	const room = await Livechat.getRoom(guest, message, roomInfo, agent);
```

Comment:
Why not just `return Livechat.getRoom(guest, message, roomInfo, agent);`?

Reply:
Fixed.